docs/docusaurus/docs/guides/connecting_to_your_data/fluent/cloud/how_to_connect_to_data_on_gcs_using_pandas.md

```diff
@@ -0,0 +1,102 @@
+
+In this guide we will demonstrate how to use Pandas to connect to data stored on Google Cloud Server.  In our examples, we will specifically be connecting to `.csv` files.  However, Great Expectations supports most types of files that Pandas has read methods for.
+
+## Prerequisites
+
+<Prerequisites>
+
+- [An installation of GX set up to work with GCS](/docs/guides/setup/optional_dependencies/cloud/how_to_set_up_gx_to_work_with_data_on_gcs.md)
+- Access to data on a GCS bucket
+- A passion for data quality
+
+</Prerequisites> 
+
+## Steps
+
+### 1. Import GX and instantiate a Data Context
+
+<ImportGxAndInstantiateADataContext />
+
+### 2. Create a Datasource
+
+We can define a GCS datasource by providing three pieces of information:
+- `name`: In our example, we will name our Datasource `"MyGcsDatasource"`
+- `bucket_or_name`: In this example, we will provide a GCS bucket
+- `gcs_options`: We can provide various additional options here, but in this example we will leave this empty and use the default values.
+
+```python title="Python code"
+datasource_name = "my_gcs_datasource"
+bucket_name = "test_docs_data"
+gcs_options = {}
+```
+
+Once we have those three elements, we can define our Datasource like so:
+
+```python title="Python code"
+datasource = context.sources.add_pandas_gcs(
+    name=datasource_name, bucket_or_name=bucket_name, gcs_options=gcs_options
+)
+```
+
+### 3. Add GCS data to the Datasource as a Data Asset
+
+```python title = "Python code"
+batching_regex = r"data/taxi_yellow_tripdata_samples/yellow_tripdata_sample_(?P<year>\d{4})-(?P<month>\d{2})\.csv"
+prefix = "data/taxi_yellow_tripdata_samples/"
+data_asset = datasource.add_csv_asset(
+    name="MyTaxiDataAsset", batching_regex=batching_regex, prefix=prefix
+)
+```
+
+<BatchingRegexExplaination storage_location_type="GCS bucket" />
+
+<!-- ## Next steps
+ 
+ <AfterCreateNonSqlDatasource /> -->
+
+## Additional information
+
+<!-- TODO: Add this once we have a script.
+### Code examples
+
+To see the full source code used for the examples in this guide, please reference the following scripts in our GitHub repository:
+- [script_name.py](https://path/to/the/script/on/github.com)
+-->
+
+<!-- ### GX Python APIs
+ 
+ For more information on the GX Python objects and APIs used in this guide, please reference the following pages of our public API documentation:
+ 
+ - `get_context(...)`
+ - `DataContext.datasources.add_pandas_gcs(...)`
+ - `Datasource.add_csv_asset(...)` -->
+
+### External APIs
+
+For more information on Google Cloud and authentication, please visit the following:
+* [gcloud CLI Tutorial](https://cloud.google.com/storage/docs/reference/libraries)
+* [GCS Python API Docs](https://googleapis.dev/python/storage/latest/index.html)
+
+### Related reading
+
+For more details regarding storing credentials for use with GX, please see our guide: [How to configure credentials](/docs/guides/setup/configuring_data_contexts/how_to_configure_credentials.md)
```

Comment:
@Rachel-Reverie This is great (subject to my minor comments above).  In addition, if I could dream for a second, I would love it if our documentation pages allowed users to copy the code and run it in their environment -- and it would be self-contained and just work, enabling users to quickly get basic end-to-end examples working.  Happy to discuss.  Thank you.

Reply:
I know we want to implement that sort of functionality in the future, but it is alas out of scope at the moment.  When we get toward that part of the docs updates I'll let you know.